pkg/operator/controllers/monitoring/monitoring_controller.go

```diff
@@ -157,13 +155,10 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&arov1alpha1.Cluster{}, builder.WithPredicates(aroClusterPredicate)).
+		Owns(&corev1.ConfigMap{}).
 		// https://github.com/kubernetes-sigs/controller-runtime/issues/1173
 		// equivalent to For(&v1.ConfigMap{}, ...)., but can't call For multiple times on one builder
-		Watches(
-			&source.Kind{Type: &corev1.ConfigMap{}},
-			&handler.EnqueueRequestForObject{},
-			builder.WithPredicates(monitoringConfigMapPredicate),
-		).
```

Comment:
I left the comments in there, but I could get rid of them as the other controllers which use `Owns` do not have this comment there.